roles/openshift_health_checker/test/aos_version_test.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import library.aos_version as aos_version
```

Comment:
This requires `library` to be a package, i.e, to have a `__init__.py` file...

Since to Ansible that's not seem as a package, but rather a bunch of modules/files, I'd go with manipulating `sys.path` and importing as:

```python
import aos_version
```

That's what we do throughout the repo for what I've seen.